filebeat/module/nginx/error/ingest/pipeline.json

```diff
@@ -6,6 +6,9 @@
       "patterns": [
         "%{DATA:nginx.error.time} \\[%{DATA:log.level}\\] %{NUMBER:process.pid:long}#%{NUMBER:process.thread.id:long}: (\\*%{NUMBER:nginx.error.connection_id:long} )?%{GREEDYDATA:message}"
       ],
+      "pattern_definitions": {
```

Comment:
just curious why we need this part?

Reply:
Without this change the -expected.json dump contained a trimmed message (just after first new line character).

Reply:
Maybe instead of redefining `GREEDYDATA`, define a new pattern called `GREEDYMULTILINE` instead?

Reply:
I believe the answer to this question is related to a different one: how familiar is thecommunity with the GREEDYDATA pattern name? This one is easy to look for using search engines and there might be already some interesting threads linked.

Also, It may be required to adjust other places in the Beats code - to preserve consistency (e.g PostgreSQL).

Just to be clear - I don't mind renaming, but wanted to highlight that this change might require more adjustments. Honestly I would keep it as is, but I'm open for a discussion :)

Reply:
We use `GREEDYMULTILINE` all over the place, I think it's a good way to differentiate the pattern from normal `GREEDYDATA` (to your question, I would expect people writing pipelines to know it). The only exception I can see is in postgresql module, which I think we should also update for consistency.

+1 on `GREEDYMULTILINE`, just to avoid confusion when using it in the grok pattern

Reply:
Fixed - changed to `GREEDYMULTILINE`.

Resolving this one.